### PR TITLE
Fix for PHP7

### PR DESCRIPTION
**Pull request by flaviocopes**
Fixes `Whoops\Exception\ErrorException: Uncaught TypeError: Argument 1 passed to Whoops\Exception\Inspector::getTrace() must be an instance of Exception, instance of Error given`

### CODE REVIEW

**Comment by staabm** [comment]
Could you add a unit Test?

**Comment by flaviocopes** [comment]
Added

File: tests/Whoops/Exception/InspectorTest.php [hunk 1]
```diff
@@ -45,6 +45,19 @@ public function testCorrectNestedFrames($value = '')
     }
 
     /**
+     * @covers Whoops\Exception\Inspector::getFrames
```
Comment (staabm) [inline tests/Whoops/Exception/InspectorTest.php]: Typo getTrace

File: tests/Whoops/Exception/InspectorTest.php [hunk 2]
```diff
@@ -45,6 +45,19 @@ public function testCorrectNestedFrames($value = '')
...
+     */
+    public function testDoesNotFailOnPHP7ErrorObject()
+    {
+        if (class_exists('Error')) {
```
Comment (staabm) [inline tests/Whoops/Exception/InspectorTest.php]: Please mark the test as skipped when Error class doesnt exist

Comment (flaviocopes) [inline tests/Whoops/Exception/InspectorTest.php]: :+1:

**Comment by denis-sokolov** [comment]
Thank you for your contribution!

**Pull request merged**